hack/e2e.sh

```diff
@@ -187,6 +187,15 @@ if [[ "test_elasticsearchcluster" = "${TEST_PREFIX}"* ]]; then
     kube_delete_namespace_and_wait "${ES_TEST_NS}"
 fi
 
+function apply_cassandracluster() {
```

Comment:
:grey_question: The other bash functions have parameters (of sorts) so it might be nice to be consistent. On the other hand it looks like we're moving away from bash based E2E tests to happy for you to leave this for now.

Reply:
And it'd be nice-to-have a check that all the environment variables that are about to be substituted are actually set.

Reply:
Let's fix these e2e things when we move to ginkgo